### PR TITLE
Enable Tox linters and Python unit tests

### PR DESCRIPTION
**Pull request by enrico-usai**
* configure logger in the main function because when importing _pcluster_dcv_authenticator_, the code was trying to initialize the _LOGGER_ by writing a file in the log dir (that is not present in the test environment)
* format code

I tested the code in a running cluster, by replacing the _pcluster_dcv_authenticator.py_ and restarting the supervisord daemon.

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    W503, # line break before binary operator => Conflicts with black style.
+    D413, # Missing blank line after last section
+# D103 Missing docstring in public function
+# E402 module level import not at top of file
+# D101 Missing docstring in public class
+# D102 Missing docstring in public method
+# D205 1 blank line required between summary line and description
+# D400 First line should end with a period
+# D401 First line should be in imperative mood
+per-file-ignores =
```
Comment (demartinofra) [inline .flake8]: this can be removed

File: .isort.cfg [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# 3 - Vertical Hanging Indent
+# from third_party import (
+#     lib1,
+#     lib2,
+#     lib3,
+#     lib4,
+# )
+multi_line_output=3
+include_trailing_comma=true
+skip=pcluster/resources/batch/custom_resources_code/crhelper
```
Comment (demartinofra) [inline .isort.cfg]: can be removed

File: .flake8 (hunk 1 above)
Comment (enrico-usai) [reply .flake8]: removed

File: .isort.cfg (hunk 2 above)
Comment (enrico-usai) [reply .isort.cfg]: done

**Pull request merged**